### PR TITLE
Made a web lab page and add web lab to the course blocks

### PR DESCRIPTION
**Pull request by caleybrock**
courses page
screen shot 2017-07-25 at 12 28 08 pm
code.org/weblab
screen shot 2017-07-25 at 12 30 40 pm

### CODE REVIEW

**Comment by caleybrock** [comment]
We're holding off on this one.

File: pegasus/sites.v3/code.org/public/educate/weblab.haml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Web Lab
+theme: responsive
+video_player: true
+---
+
+%link{href: "/css/jumbotron-banner.css", rel: "stylesheet"}
+%link{href: "/shared/css/course-blocks.css", rel: "stylesheet"}
+
+= view :jumbotron_banner, img: CDO.code_org_url("/images/app-lab/girl-cropped.jpg"), title: "Web Lab (beta)", desc: "Web Lab is a programming environment where you can make simple web pages using HTML and CSS. Design your web pages and share your site in seconds.", specs: "Ages 13+, all modern browsers, English only", url: CDO.studio_url('/p/weblab'), cta: "Make an web page", photo_cred: "Photo courtesy of Fundación Sadosky", nav_bar: [{text: 'Resources', url: '#resources'}]
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/weblab.haml:10]: I wonder if we can put this inside a `:ruby` block and use multiple lines?

File: pegasus/sites.v3/code.org/public/educate/weblab.haml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+#resources.responsive-body
+  %h1 Resources
+  %h2 Try the Computer Science Discoveries unit that introduces Web Lab
+  %h3= I18n.t("csd2_name")
+  %h4 Grades 7 - 9
+  .smalltext In Unit 2, students are empowered to create and share the content on their own web pages. They begin by thinking about the role of the web, and how it can be used as a medium for creative expression. As students develop their pages and begin to see themselves as programmers, they are encouraged think critically about the impact of sharing information online and how to be more critical content consumers. They are also introduced to problem solving as it relates to programming, as they learn valuable skills such as debugging, commenting, and structure of language. At the conclusion of the unit, students compile their work to create a personal website they can publish and share.
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/weblab.haml:17]: nit: very long line

File: pegasus/sites.v3/code.org/public/educate/weblab.haml (hunk 1 above)
Comment (caleybrock) [reply pegasus/sites.v3/code.org/public/educate/weblab.haml:10]: good thought, I'm making a few pages that are all like this, and might do this as a follow up to all.

**Pull request merged**